holidays/countries/yemen.py

```diff
@@ -0,0 +1,138 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars import _CustomIslamicHolidays
+from holidays.calendars.gregorian import (
+    MAR,
+    APR,
+    MAY,
+    JUN,
+    JUL,
+    AUG,
+    SEP,
+    OCT,
+)
+from holidays.groups import InternationalHolidays, IslamicHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class Yemen(HolidayBase, InternationalHolidays, IslamicHolidays):
+    """Yemen holidays.
+
+    References:
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Yemen>
+        * <https://yemenembassy.it/festivities/>
+    """
+
+    country = "YE"
+    default_language = "ar"
+    # %s (estimated).
+    estimated_label = tr("%s (مُقدَّر)")
+    start_year = 1990
```

Comment:
```suggestion
    # The Republic of Yemen was declared on 22 May 1990.
    start_year = 1991
```
Perhaps we will change the year to a later one, according to the available sources of information about the holidays.